test/jdk/java/awt/Window/OwnedWindowsLeak/OwnedWindowsLeak.java

```diff
@@ -70,31 +71,22 @@ public static void main(String[] args)
         // Third, make sure all the weak references are null
         for (WeakReference<Window> ref : children)
         {
-            if (ref.get() != null)
-            {
-                throw new RuntimeException("Test FAILED: some of child windows are not GCed");
+            while (ref.get() != null) {
```

Comment:
Will wait till the end of the test and log what window is leaked. If it fails then it will be 100% product bug.